test/specs/linters/attribute_quotes.js

```diff
@@ -61,34 +57,29 @@ describe('lesshint', function () {
                 }
             };
 
-            ast = linter.parseAST(source);
+            ast = parseAST(source);
             ast = ast.first().first('selector').first('simpleSelector');
 
-            actual = attributeQuotes({
-                config: options,
-                node: ast,
-                path: 'test.less'
-            });
+            actual = lint(options, ast);
 
             assert.deepEqual(actual, expected);
         });
 
         it('should return null for value-less selectors', function () {
```

Comment:
Change all instances of `should return null [...]` to `should return undefined [...]` (or something like that).